.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,15 @@
+// For format details, see https://aka.ms/devcontainer.json
+{
+	"name": "pallets/flask",
+
+	// Features to add to the devcontainer
+	"features": {
+		"ghcr.io/devcontainers/features/python:1": {}
```

Comment:
This installs Python and basic utilities. For more info, see https://github.com/devcontainers/features/tree/main/src/python